ifbcat_api/model/tool/database.py

```diff
@@ -0,0 +1,205 @@
+import json
```

Comment:
File is not in the right place, should not be in `tool/` directory

Reply:
> File is not in the right place, should not be in `tool/` directory

Moved to `/model/`